mingw-w64-highfive/PKGBUILD

```
@@ -0,0 +1,70 @@
+# Maintainer: Václav Šmilauer <eu@doxos.eu>
+
+_realname=highfive
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=2.3.1
+pkgrel=1
+pkgdesc='Modern header-only C++11 friendly interface for libhdf5.'
```

```suggestion
pkgdesc='Modern header-only C++11 friendly interface for libhdf5 (mingw-w64)'
```
To silent the warning